ownership-chain/precompile/evolution-collection/src/lib.rs

```diff
@@ -79,6 +81,8 @@ where
 			.map_err(|_| revert("invalid collection address"))?;
 
 		if let Some(owner) = LaosEvolution::<Runtime>::collection_owner(collection_id) {
+			handle.record_cost(GasCalculator::<Runtime>::db_read_gas_cost(1))?;
```

Comment:
wouldn't be better add benchmarking to `collection_owner` call to follow the same approach as in `EvolutionCollectionFactory` or it doesn't make sense to add benchmarking to a function that only reads from storage?

Reply:
altough after rethinking.. why read from evm has a cost?

Reply:
it has a cost on the block, i.e there is one read in DB. I know it doesn't make sense from EVM POV, but we still need to record this since it goes through precompiles